src/ListContainer.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const flexContainerGallery = {
```

Comment:
Should look into CSS Modules or `emotion` CSS, because these inject the CSS inline which doesn't lend itself to a great developer experience when inspecting and tweaking values to see their effect, can't use pseudo classes, can't use media queries, can't animate... And it's just generally frowned upon.

Reply:
would those have to be bundled with my dist? 
trying to do as much "vanilla react" as possible with very very little dependence on outside libraries. 
I'm willing to give a look tho if it doesn't damage my requirements.

Reply:
Yeah, Webpack would create CSS files from `emotion` or CSS Modules, which would be bundled with your dist. But to my knowledge, your consumers won't need to install them as dependencies to use your components, as your components would reference the css files generated at build.

Reply:
nice i'll def look into it then!